### ISSUE TITLE
Getting error Error: cannot connect to Tiller

### ISSUE THREAD

**Issue by ramakila** [helm/helm#4733]
Output of `helm version`: 
[root@edaticpmanage ~]# helm version
Client: &version.Version{SemVer:"v2.7.3+icp", GitCommit:"27442e4cfd324d8f82f935fe0b7b492994d4c289", GitTreeState:"dirty"}
Error: cannot connect to Tiller

Output of `kubectl version`:

[root@edaticpmanage ~]# kubectl version
Client Version: version.Info{Major:"1", Minor:"10", GitVersion:"v1.10.5", GitCommit:"32ac1c9073b132b8ba18aa830f46b77dcceb0723", GitTreeState:"clean", BuildDate:"2018-06-21T11:46:00Z", GoVersion:"go1.9.3", Compiler:"gc", Platform:"linux/ppc64le"}
Server Version: version.Info{Major:"1", Minor:"10", GitVersion:"v1.10.0+icp", GitCommit:"1c6988c2361d678dab8ea5ddfab82cb52680b20b", GitTreeState:"clean", BuildDate:"2018-04-27T05:47:17Z", GoVersion:"go1.9.3", Compiler:"gc", Platform:"linux/ppc64le"}

Cloud Provider/Platform (AKS, GKE, Minikube etc.): 
ICP 2103/PPC64

**Comment by hickeyma** [comment]
Did you run `helm init` after installing or `helm init --upgrade` after upgrading?

**Comment by jascott1** [comment]
Hi @ramakila 

Seems like a connection problem. Can you see the status of the Tiller pod with `kubectl get pod -n kube-system` ?

**Comment by ramakila** [comment]
Yes @jascott1.

[root@edaticpmanage ~]# kubectl get pod -n kube-system  | grep -i tiller
tiller-deploy-57b4bf59d-rrkxg                                 1/1       Running   0          7d
[root@edaticpmanage ~]# helm version
Client: &version.Version{SemVer:"v2.7.3+icp", GitCommit:"27442e4cfd324d8f82f935fe0b7b492994d4c289", GitTreeState:"dirty"}
Error: cannot connect to Tiller
[root@edaticpmanage ~]#

**Comment by hickeyma** [comment]
Can you try: `helm init --upgrade` ?

**Comment by jascott1** [comment]
Hi @ramakila Sorry for the delay but did you find a cause and solution?

**Comment by ramakila** [comment]
Yes . Thank you. we have ICP 3.1.1 on bigger machines now and everything is fine.

**Comment by gurunathsane** [comment]
Still facing the same issue with ICP 3.10

**Comment by kaushalrai7797** [comment]
Hi! I am facing this issue. Helm init says error installing: server could not find the requested resource. Helm version command shows error could not find tiller

**Comment by bacongobbler** [comment]
@kaushalrai7797 have you checked #6374?

**Comment by kevinsingapore** [comment]
i have also meet the same error.
 
kuberctl v1.15.3
helm v2.15.1

**Comment by kevinsingapore** [comment]
[root@kvm147 helm]# helm version
Client: &version.Version{SemVer:"v2.15.1", GitCommit:"cf1de4f8ba70eded310918a8af3a96bfe8e7683b", GitTreeState:"clean"}
Error: cannot connect to Tiller
[root@kvm147 helm]#

**Comment by kevinsingapore** [comment]
[root@kvm147 helm]# kubectl get pods -n kube-system -l name=tiller
NAME                             READY   STATUS    RESTARTS   AGE
tiller-deploy-6874cfcfbb-bg449   1/1     Running   0          3m25s
[root@kvm147 helm]#

**Comment by MarkRosen** [comment]
Please consider specifying the namespace where tiller is running.
For example:
```$ helm version
Client: &version.Version{SemVer:"v2.9.1", GitCommit:"20adb27c7c5868466912eebdf6664e7390ebe710", GitTreeState:"clean"}
Error: cannot connect to Tiller

$ helm --tiller-namespace mynamespace version
Client: &version.Version{SemVer:"v2.9.1", GitCommit:"20adb27c7c5868466912eebdf6664e7390ebe710", GitTreeState:"clean"}
Server: &version.Version{SemVer:"v2.12.3", GitCommit:"eecf22f77df5f65c823aacd2dbd30ae6c65f186e", GitTreeState:"clean"}
```

**Comment by kevinsingapore** [comment]
> Please consider specifying the namespace where tiller is running.
> For example:
> 
> ```
> $ helm version
> Client: &version.Version{SemVer:"v2.9.1", GitCommit:"20adb27c7c5868466912eebdf6664e7390ebe710", GitTreeState:"clean"}
> Error: cannot connect to Tiller
> 
> $ helm --tiller-namespace mynamespace version
> Client: &version.Version{SemVer:"v2.9.1", GitCommit:"20adb27c7c5868466912eebdf6664e7390ebe710", GitTreeState:"clean"}
> Server: &version.Version{SemVer:"v2.12.3", GitCommit:"eecf22f77df5f65c823aacd2dbd30ae6c65f186e", GitTreeState:"clean"}
> ```
> 
> Alternatively, you can export TILLER_NAMESPACE (as per #2519 )
> 
> ```
> $ export TILLER_NAMESPACE=mynamespace
> $ helm  version
> Client: &version.Version{SemVer:"v2.9.1", GitCommit:"20adb27c7c5868466912eebdf6664e7390ebe710", GitTreeState:"clean"}
> Server: &version.Version{SemVer:"v2.12.3", GitCommit:"eecf22f77df5f65c823aacd2dbd30ae6c65f186e", GitTreeState:"clean"}
> ```

i had upgrade the helm version to v3.0.0
v3 does not need tiller.
